app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2672,11 +2688,16 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
     out.source.on_edge_drop <+ overlapping_edges;
 
     drop_on_bg_up  <- background_up.gate(&connect_drag_mode);
-    drop_edges     <- any (drop_on_bg_up,touch.background.down);
-    edge_to_drop_without_targets <= drop_edges.map(f_!(model.take_edges_with_detached_targets()));
-    edge_to_drop_without_sources <= drop_edges.map(f_!(model.take_edges_with_detached_sources()));
-    edge_to_drop <- any(edge_to_drop_without_targets,edge_to_drop_without_sources);
-    eval edge_to_drop ((id) model.remove_edge(id));
+    drop_edges     <- any (drop_on_bg_up,click_to_drop_edge);
+
+    edge_dropped_to_create_node <= drop_edges.map(f_!(model.edges_with_detached_targets()));
+    out.source.on_edge_drop_to_create_node <+ edge_dropped_to_create_node;
```

Comment:
Do you need to connect it here to `out....`? We try to connect to out minimal amounts of things, and there was already a logic that was opening the searcher – are you not able to plug into this existing logic here? Maybe not, but it's a little bit hard for me to understand what this code does because its hard to understand the var names now.

Reply:
The searcher is opened when the node enters `being_edited` state - but the Project View is creating new nodes and is starting editing them, not the Graph View. So there is no way to open searcher from the Graph View at the moment, I need an additional output to signal Project View.
Perhaps there is a way to refactor the code in such a way that Graph View would be responsible for node creation, but it feels like quite a big change.